test/utils/pickers/describeValue/testControlledUnControlled.tsx

```diff
@@ -259,7 +257,10 @@ export const testControlledUnControlled: DescribeValueTestSuite<any, any> = (
         expect(fieldRoot).to.have.class(inputBaseClasses.error);
         expect(fieldRoot).to.have.attribute('aria-invalid', 'true');
 
-        if (params.type === 'date-range' && !params.isSingleInput) {
+        if (
+          (params.type === 'date-range' || params.type === 'date-time-range') &&
+          !params.isSingleInput
+        ) {
```

Comment:
```suggestion
        if (isRangeType && !params.isSingleInput) {
```

Reply:
Sadly, can't do it.
TS does not resolve the `params` type correctly in that case. 🙈 
<img width="1022" alt="Screenshot 2024-02-28 at 16 25 52" src="https://github.com/mui/mui-x/assets/4941090/d444f874-7f55-4a22-a89e-02c7e31dabff">

Reply:
eh ... but that wasn't my proposed change! 
`params.type === 'date-range' || params.type === 'date-time-range'` is the same as `isRangeType`, which gets defined in line 31

Reply:
How is it not the same proposed change? 🤔 
The problem is that TS can't infer that `isSingleInput` exists on `params` with that kind of check 🙈 
I've looked into it an will replace the `[].includes` check for `=== || ===` to make it work. 👌

Reply:
oh, you meant your change? I was referring to my suggestion! :D